store/errors.go

```diff
@@ -49,6 +49,8 @@ var (
 
 	// ErrInvalidCredentials is returned on login error
 	ErrInvalidCredentials = errors.New("invalid credentials")
+
+	ErrTosNotAccepted = errors.New("TOS not accepted")
```

Comment:
It looks like this is a string that eventually surfaces to a end-user. If this is the case I think we want to write it a bit friendlier like `Terms of service not accepted`.